aemiers/rancid-tomatillos
# Feature/responsive

## Pull Request

### Description

* Is this a feature or a fix?
Feature
* Where should the reviewer start?
Review the media queries at the bottom of MovieList, MovieCard, MovieDetails, Header and Form
* What functionalities do these changes affect? Why were these changes made? What was the motivation behind these changes?
This makes the site responsive for a variety of devices. 
* What outside features or research did you use to implement this fix?
Nope
* How should this be tested?
Open the app with DEV tools and switch the responsive view. Resize the viewing window to see the breakpoints. 

***

#### Please review considerations below:
